README.md

```diff
@@ -76,3 +76,48 @@ Solution:
 3. Change related ip of host.docker.internal and gateway.docker.internal to your new ip (found in terminal using the ipconfig command: e.g. 192.168.0.1)
 4. save
 5. restart the application.
+
+## Adding an ADR Algorithm
+When the ADR Algorithm has been tested, and is ready for deployment, it has to be built as a Go Module.
+
+The module is created running:
+```bash
+mkdir example-mod-name`.
+cd example-mod-name`.
+go mod init
```

Comment:
Aren't you missing the name of the module after `init`?

Reply:
You should be able to do it without, but to make it more general I have added it.

Reply:
You're right. It error'd out when I attempted it. It might have had something to do with path not being registered in Go